examples/plotting/file/pixel_fun.py

```diff
@@ -0,0 +1,37 @@
+#Just for fun, taking a jpg image and plotting it pointillistic style on a bokeh canvas using circle glyphs!
+#Requires PIL!
+import Image
```

Comment:
This example is failing on Travis because it does not get the `Image` module from PIL, we have two options... we add PIL (or better Pillow) to our test engine or we skip this example from testing... thoughts?